tools/runner/runner.go

```diff
@@ -145,6 +145,7 @@ func (r *Runner) runTest(ctx context.Context, config *grpcv1.LoadTest, reporter
 			if err != nil {
 				reporter.Error("Could not save pod logs: %s", err)
 			}
+			reporter.AddProperty("Load test name", loadTest.Name)
```

Comment:
Wondering if we want to use LoadTest name. :)

Reply:
Yes, that sounds good :)